src/tests/ftest/recovery/cat_recov_core.py

```diff
@@ -0,0 +1,30 @@
+"""
+  (C) Copyright 2024 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from daos_core_base import DaosCoreBase
+
+
+class CatRecovCoreTest(DaosCoreBase):
+    # pylint: disable=too-many-ancestors,too-many-public-methods
+    """Runs daos catastrophic recovery core tests.
+
+    :avocado: recursive
+    """
+
+    def test_daos_cat_recov_core(self):
+        """Jira ID: DAOS-13047
+
+        Test Description:
+            Run daos_test -F
+
+        Use cases:
+            Core tests for daos catastrophic recovery
+
+        :avocado: tags=all,daily_regression
+        :avocado: tags=hw,medium,provider
+        :avocado: tags=recovery,test_cat_recovery,cat_recov_core
```

Comment:
We shouldn't use tags that start with `test_` unless it's the name of the test method. Can we just call it `cat_recovery`? 

To be consistent, I think `cat_recov_core` should be `cat_recovery_core`

Also, do we really need three separate tags? `recovery`, `cat_recovery`, `cat_recovery_core`?

The test method name tag is already unique amongst all tests: `test_daos_cat_recov_core`


And all the tests being tagged with `test_cat_recovery` now already have `recovery`

Reply:
I will unify the tag as "cat_recov", remove "test_cat_recovery" and "cat_recovery_core".